patches/denymount+2.2.0.patch

```diff
@@ -1,38 +0,0 @@
-diff --git a/node_modules/denymount/lib/index.js b/node_modules/denymount/lib/index.js
```

Comment:
This patch was no longer necessary with the `denymount` version bump from 2.2.0 to 2.3.0.